packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,22 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Button
+  class={{this.classNames}}
+  @size={{this.size}}
+  @isFullWidth={{@isFullWidth}}
+  @text={{@text}}
+  @icon={{this.icon}}
+  @isIconOnly={{@isIconOnly}}
+  @color="secondary"
+  @iconPosition="trailing"
+  {{clipboard
+    text=this.textToCopy
+    target=this.targetToCopy
+    container=this.container
```

Comment:
We could've probably used the arguments directly without defining getters for each one as we don't seem to be altering these values or returning any custom defaults. However it may be a good way to document these arguments.

```suggestion
    text=@textToCopy
    target=@targetToCopy
    container=@container
```

Reply:
+1 I would remove those getters to simplify the code if feasible

Reply:
@KristinLBradley @alex-ju this was really just so we could put this into our docs, otherwise the consumer would definitely have to leave our site just to figure out the API for the things they really need to use. I'd prefer to not do that, but willing to align with what the group wants. WDYT?

Reply:
Ok after talking it through a bit more, am fine with the change. 👍